enterprise/tools/upload_local_snapshot/upload_local_snapshot.go

```diff
@@ -29,14 +29,48 @@ import (
 	bspb "google.golang.org/genproto/googleapis/bytestream"
 )
 
-// This tool should be run on the executor with the desired local snapshot in
+// This tool should be run on the executor pod with the desired local snapshot in
 // its filecache. It will upload the local snapshot to the remote cache,
 // so it can be more easily inspected and run with vmstart, for example.
+//
+// Steps:
+// 1. SSH into the executor pod the local snapshot is on.
+//		a. From the action details page of the invocation link, find the executor host ID
+// 		b. Search for the executor host ID in the `Executors` tab
+//			- Likely will be under the "Shared Executors" group
+//		c. Grab the IP address of the executor
+// 		d. Find the pod name associated with that IP address
+//			- kubectl --namespace executor-prod get pods -o wide
+//			- Search for the IP
+// 		e. SSH into the pod
+//			-  kubectl --namespace=executor-prod exec -it executor-XXX /bin/bash
+//
+// 2. Clone the buildbuddy repo on the executor so you can use this tool
+//		a. git clone https://github.com/buildbuddy-io/buildbuddy.git
+//
+// 3. Download bazel to run this tool
+//		a. curl -Lo /usr/local/bin/bazelisk https://github.com/bazelbuild/bazelisk/releases/download/v1.15.0/bazelisk-linux-amd64
+//		b. chmod +x /usr/local/bin/bazelisk
```

Comment:
To avoid increasing the workload on the executor, another option is to build the tool on a Linux machine using `--config=static` then use `kubectl cp --namespace=executor-prod bazel-bin/enterprise/tools/upload_local_snapshot/upload_local_snapshot_/upload_local_snapshot executor-XXX:/tmp/upload_local_snapshot`